src/components/Hero/ContentMedia/index.tsx

```diff
@@ -12,47 +13,44 @@ import { Page } from '@root/payload-types'
 import classes from './index.module.scss'
 
 export const ContentMediaHero: React.FC<
-  Pick<Page['hero'], 'richText' | 'media' | 'mediaWidth' | 'links'> & {
+  Pick<Page['hero'], 'richText' | 'media' | 'links' | 'sidebarContent'> & {
     breadcrumbs?: Page['breadcrumbs']
   }
-> = ({ richText, media, mediaWidth, breadcrumbs, links }) => {
+> = ({ richText, media, links, sidebarContent }) => {
```

Comment:
@paulpopus You removed the usage of the `mediaWidth` prop right? Is it confirmed that this hero will only have the one media width? 

If so, then this prop needs to be removed from the cms side and then the payload types need to be updated in the cms and here.